email_alert_service/models/update_subscriber_list.rb

```diff
@@ -0,0 +1,56 @@
+class UpdateSubscriberList
+  def initialize(document, logger)
+    @document = document
+    @logger = logger
+  end
+
+  def trigger
+    get_subscriber_list
+    return unless subscriber_list_slug
+
+    if updateable_parameters.empty?
+      logger.info "Update cancelled for subscriber list: #{subscriber_list_slug}, no updatable paramters found"
+      return
+    end
```

Comment:
Based on the definition of `updateable_parameters`, this condition will never be true

Reply:
humm yeah, I think this can go - if we outsouricng validation to the API makes sense to let this get caught in the other 422